.github/pull_request_template.md

```diff
@@ -10,23 +10,21 @@
 <!-- A few sentences describing the overall goals of the pull request's commits. -->
 
 ## Related Issue(s)
-<!-- List related issues and pull requests: -->
+<!-- List related issues and pull requests. If this PR fixes an issue, please add it using Fixes #????  -->
 
-- 
 
 ## Checklist
-- [ ] Should this PR be backported?
 - [ ] Tests were added or are not required
 - [ ] Documentation was added or is not required
 
 ## Deployment Notes
 <!-- Notes regarding deployment of the contained body of work. These should note any db migrations, etc. -->
 
-## Impacted Areas in Vitess
-Components that this PR will affect:
+<!-- Labeling PRs
 
-- [ ]  Query Serving
-- [ ]  VReplication
-- [ ]  Cluster Management
-- [ ]  Build/CI
-- [ ]  VTAdmin
+For each PR, try to find one `Type:` label and one `Component:` label. 
+This will help in finding a reviewer, and in creating release notes for the next release.
+
+If your PR needs to be backported, add the label `Backport me!`.
+
+-->
```

Comment:
Please consider that only people with `Write` privileges on our GitHub repo, are able to create labels. Which means the casual contributor is unable to classify their PR by labeling.

I'm not sure if this undermines the purpose of this PR...

Reply:
I don't think it does. We as maintainers should be able to handle labelling before PRs get merged.
And about backporting - deciding what gets backported and what does not has in my experience always been a discussion between maintainers. 
Maybe we should just remove this comment altogether from the PR template.